nimbus.py

```diff
@@ -16,24 +16,34 @@ def __init__(self):
         self.qa_dict = create_qa_mapping(
             generate_fact_QA("q_a_pairs.csv")
         )
+        self.nimbus_nlp = NimbusNLP()
 
     def answer_question(self, question):
-        ans_dict = NIMBUS_NLP.predict_question(question)
+        ans_dict = self.nimbus_nlp.predict_question(question)
         print(ans_dict)
         try:
             qa = self.qa_dict[ans_dict["question class"]]
         except KeyError:
+            # Printed if question isn't found. This occurs because the training set is broader
+            # than the answerable question set.
             return "I'm sorry, I don't understand. Please try another question."
         else:
             answer = qa.answer(ans_dict)
             if answer is None:
+                # Printed when a database query was made and a null value was returned.
+                # Should be handled in the QA class in the future.
```

Comment:
so glad these comments were included! It will help greatly when debugging in the future! Thanks @cameron-toy !